platform/assert-web.js

```diff
@@ -7,6 +7,7 @@
 
 export default function assert(test, message) {
   if (!test) {
+    debugger;
```

Comment:
please, remove this.

Reply:
Actually I was talking with Doug about leaving this in. A lot of our exceptions get swallowed in the app-shell (because of Promises and whatnot), and this makes it much easier to find out what's happening. It doesn't have any effect on the code unless the debugger is open so it doesn't impact normal user experience.